catalogue/webapp/pages/item.js

```diff
@@ -293,6 +293,43 @@ const ItemPage = ({
       imageAltText={''}
       hideNewsletterPromo={true}
     >
+      <Layout12>
+        <div
+          className={classNames({
+            [spacing({ s: 4 }, { margin: ['bottom'] })]: true,
+            [spacing({ s: 6 }, { padding: ['top'] })]: true,
+          })}
+        >
+          <h1
+            className={classNames({
+              [font({ s: 'HNM3', m: 'HNM2', l: 'HNM1' })]: true,
+            })}
+            style={{
+              whiteSpace: 'nowrap',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+            }}
+            title={title}
+          >
+            {title}
+          </h1>
+          <NextLink
```

Comment:
Why do we cut out the page and query when we go back?

Reply:
Oversight. Will update.